<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>810</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>810</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>My myzzle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
border-image: url(:/picture/background.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>610</width>
      <height>610</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>351</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="Button_reselection">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,153,51);
border-style:outset;                
border-width:7px;                 
border-radius:10px;                 
border-color:rgba(255,255,255,100);
font:bold 22px ;
color:rgba(0,0,0,100);
padding:6px;
                   </string>
         </property>
         <property name="text">
          <string>难度重选</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="Button_original">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,153,51);
border-style:outset;                
border-width:7px;                 
border-radius:10px;                 
border-color:rgba(255,255,255,100);
font:bold 22px ;
color:rgba(0,0,0,100);
padding:6px;
                   </string>
         </property>
         <property name="text">
          <string>查看原图</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Button_upset">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,153,51);
border-style:outset;                
border-width:7px;                 
border-radius:10px;                 
border-color:rgba(255,255,255,100);
font:bold 22px ;
color:rgba(0,0,0,100);
padding:6px;
                   </string>
         </property>
         <property name="text">
          <string>图片重排</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Button_records">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,153,51);
border-style:outset;                
border-width:7px;                 
border-radius:10px;                 
border-color:rgba(255,255,255,100);
font:bold 22px ;
color:rgba(0,0,0,100);
padding:6px;
                   </string>
         </property>
         <property name="text">
          <string>本地记录</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Button_chooseimg">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,153,51);
border-style:outset;                
border-width:7px;                 
border-radius:10px;                 
border-color:rgba(255,255,255,100);
font:bold 22px ;
color:rgba(0,0,0,100);
padding:6px;
                   </string>
         </property>
         <property name="text">
          <string>图片重选</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Button_local_img">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,153,51);
border-style:outset;                
border-width:7px;                 
border-radius:10px;                 
border-color:rgba(255,255,255,100);
font:bold 22px ;
color:rgba(0,0,0,100);
padding:6px;
                   </string>
         </property>
         <property name="text">
          <string>添加新图</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="RButton_click">
         <property name="text">
          <string>点击模式</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RButton_drag">
         <property name="text">
          <string>拖动模式</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 5px solid  #996633;</string>
         </property>
         <property name="text">
          <string>当前用时:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_time">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 5px solid  #996633;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_steps">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 5px solid  #996633;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 5px solid  #996633;</string>
         </property>
         <property name="text">
          <string>当前步数:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_picture</zorder>
   <zorder>RButton_click</zorder>
   <zorder>RButton_drag</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
